src/lib/Scenes/Artwork/Components/FollowArtistButton.tsx

```diff
@@ -0,0 +1,88 @@
+import { Sans } from "@artsy/palette"
+import { FollowArtistButton_artist } from "__generated__/FollowArtistButton_artist.graphql"
+import { FollowArtistButtonMutation } from "__generated__/FollowArtistButtonMutation.graphql"
+import React from "react"
+import { TouchableWithoutFeedback } from "react-native"
+import { commitMutation, createFragmentContainer, graphql, RelayProp } from "react-relay"
+
+interface Props {
+  artist: FollowArtistButton_artist
+  relay: RelayProp
+}
+
+interface State {
+  isFollowedChanging: boolean
+}
+
+export class FollowArtistButton extends React.Component<Props, State> {
+  state = { isFollowedChanging: false }
```

Comment:
Seems like `isFollowedChanging` isn't being read anywhere, only written to. Can we delete it?